libr/bin/Makefile

```diff
@@ -27,7 +27,6 @@ OBJS+=mangling/demangler.o
 OBJS+=mangling/microsoft_demangle.o
 OBJS+=mangling/objc.o mangling/cxx.o mangling/msvc.o
 OBJS+=mangling/swift.o mangling/swift-sd.o
-OBJS+=mangling/dlang.o
```

Comment:
Why?

Reply:
It seems it's not used anywhere... maybe it was just removed in one of the recent related commit?

Reply:
yeah its fine to kill i t, its now in blang.c